apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -76,9 +76,10 @@ export class FilesStorageUC {
 	}
 
 	private createFileDescription(file: internal.Readable, info: busboy.FileInfo, req: Request): IFile {
+		const name = Buffer.from(info.filename, 'binary').toString();
```

Comment:
'binary' is an alias for 'latin1': https://nodejs.org/api/buffer.html#buffers-and-character-encodings
but 'binary' sounds misleading. better use 'latin1'.